src/js/providers/html5.js

```diff
@@ -837,7 +838,25 @@ define([
         this.getName = function() {
             return { name : _name };
         };
+        this.setCurrentAudioTrack = _setCurrentAudioTrack;
 
+        function _setAudioTracks(tracks) {
+            if(tracks && tracks.length > 0) {
+                _currentAudioTrackIndex = 0;
+                tracks[_currentAudioTrackIndex].enabled = true;
```

Comment:
So instead look for the first enabled track, and set `_currentAudioTrackIndex ` to that.